src/bin/pg_autoctl/cli_enable_disable.c

```diff
@@ -1180,9 +1180,20 @@ update_monitor_connection_string(KeeperConfig *config)
 		return false;
 	}
 
+	char scrubbedConnectionString[MAXCONNINFO] = { 0 };
+	if (parse_and_scrub_connection_string(newPgURI, scrubbedConnectionString))
+	{
+		log_info("Trying to connect to monitor using connection string \"%s\"",
+				 scrubbedConnectionString);
+	}
+	else
+	{
+		log_info(
```

Comment:
I now notice we return false in that case. I suppose we should then either `log_error` or maybe `log_warn` if appropriate (not sure it would be).